build/build.cake

```diff
@@ -154,16 +242,12 @@ Task("Publish")
         ApiKey = apiKey,
         Source = apiUrl
     });
-}).OnError(exception =>
-{
-    Information("Publish Task failed, but continuing with next Task...");
-    publishingError = true;
 });
 
 
 Task("Upload-Coverage-Report")
     .WithCriteria(() => FileExists(paths.Files.TestCoverageOutput))
-    .WithCriteria(() => !parameters.IsLocalBuild)
+    .WithCriteria(() => parameters.IsRunningOnWindows && !parameters.IsLocalBuild)
```

Comment:
Even though coverage is calculated for each job, there is no point of publishing it twice - that is why I publish it only from windows job/machine

Reply:
I wonder if this should be a parameter we specify in each build instead? (rather than relying on IsRunningOnWindows for that flag) Then we can use build parameters on either build to make that publish coverage.

Reply:
That can be parameter for sure. Let me just check if I can find lib/tool for uploading to coveralls io from linux.

Reply:
@dtchepak switched to https://www.nuget.org/packages/coveralls.net/1.0.0 package for uploading coverate reports. Also `UploadCoverageReport` was added which controls if the report should be uploaded (the flag is valid only if the build is run on CI, we dont want to upload coverages for local builds)